packages/engine-core/src/data-proxy/DataProxyEngine.ts

```diff
@@ -129,14 +136,21 @@ export class DataProxyEngine extends Engine {
     return this.requestInternal<T>({ query, variables: {} }, headers, attempt)
   }
 
-  async requestBatch<T>(queries: string[], headers: Record<string, string>, isTransaction = false, attempt = 0) {
+  async requestBatch<T>(
+    queries: string[],
+    headers: Record<string, string>,
+    transaction?: BatchTransactionOptions,
+    attempt = 0,
+  ) {
+    const isTransaction = Boolean(transaction)
```

Comment:
This looks like it can be moved directly into the object

Reply:
It's also used in a log message below